includes/Admin/Product_Categories.php

```diff
@@ -143,10 +148,30 @@ public function get_google_product_category_field_title() {
 	 *
 	 * @since 2.1.0-dev.1
 	 *
-	 * @return string
+	 * @param int $term_id term ID
+	 * @param int $tt_id term taxonomy ID
+	 * @param string $taxonomy Taxonomy slug
 	 */
-	public function save_google_product_category() {
+	public function save_google_product_category( $term_id, $tt_id, $taxonomy ) {
+
+		$google_product_category_id = Framework\SV_WC_Helper::get_posted_value( self::FIELD_GOOGLE_PRODUCT_CATEGORY_ID );
+
+		\SkyVerge\WooCommerce\Facebook\Product_Categories::update_google_product_category_id( $term_id, $google_product_category_id );
+
+		$term = get_term( $term_id, $taxonomy, ARRAY_A );
```

Comment:
We should check that `get_term()` returned an array instead of `null` or `WP_Error`.

Reply:
Updated in https://github.com/facebookincubator/facebook-for-woocommerce/pull/1551/commits/c6bf98999488a4f950272fd83249940ab5b32c74.